pyro/optim/adagrad_rmsprop.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.optim.optimizer import Optimizer
+
+
+class AdagradRMSProp(Optimizer):
+    """
+    Implements a mash-up of the Adagrad algorithm and RMSProp. See ref. [1].
```

Comment:
Is it worth adding the update equation to the docstring?

Reply:
instead i'll put a more direct reference in the docstring